File: app/src/main/java/com/botijasoftware/utils/materials/passoptions/PassOptionMaskColor.java
package com.botijasoftware.utils.materials.passoptions;

import android.opengl.GLES10;

import com.botijasoftware.utils.materials.Material;

public class PassOptionMaskColor extends PassOption {

	@Override
	public void set(Material material) {
		GLES10.glColorMask(false, false, false, false);
	}

	@Override
	public void unSet() {

		GLES10.glColorMask(true, true, true, true);
	}

}

File: app/src/main/java/com/botijasoftware/utils/materials/passoptions/PassOptionMaskColor.kt
package com.botijasoftware.utils.materials.passoptions

import android.opengl.GLES10

import com.botijasoftware.utils.materials.Material

class PassOptionMaskColor : PassOption() {

    override fun set(material: Material) {
        GLES10.glColorMask(false, false, false, false)
    }

    override fun unSet() {

        GLES10.glColorMask(true, true, true, true)
    }

}
